### PR TITLE
Layout like outlook

### PR DESCRIPTION
**Pull request by adeclosset**
Creating the PR to make the changes easier to track

### CODE REVIEW

**Comment by octaveauger** [comment]
@adeclosset you'll want to git pull again - and now there should be no JS issue anymore :)

File: app/views/authorisations/_email.html.erb [hunk 1]
```diff
@@ -1,7 +1,12 @@
 <div class="panel-group">
   <div class="panel panel-default">
     <div class="panel-heading <%= 'collapse' if email_counter == 0 %>" data-role="toggle-collapse" data-id="<%= email.message_id %>">
-      <div class="col-md-9">
+      <div class="col-md-1">
+        <% if !@message_attachments.find_for_message(email.message_id).empty? %>
```
Comment (adeclosset) [inline app/views/authorisations/_email.html.erb]: @octaveauger - just need your help to not show the attachment when it's inline

File: app/views/authorisations/_email.html.erb [hunk 2]
```diff
@@ -14,64 +19,62 @@
...
+                   <div id="expanded-<%= email.message_id %>">
+                    <%= email.body_html_sections[:expanded].html_safe %>
+                   </div>
+                </div>
               <% end %>
-            </div>
+            <% end %>
           </div>
+        </div>
+        <% if !@message_attachments.find_for_message(email.message_id).empty? %>
```
Comment (adeclosset) [inline app/views/authorisations/_email.html.erb]: Same here on attachment

**Comment by octaveauger** [comment]
@adeclosset Lots of small improvements. What remains to be done (in my opinion):
- The email loader gif is not great (you can replace ajax-loader-big if you find a better one)
- The attachment symbol in the thread list should be below the pin
  Anything else? Then we can merge.

**Comment by octaveauger** [comment]
@adeclosset I'm merging this now - to be deployed once my work on the sync bug fix is done

**Pull request merged**